src/web/deployment.md

```diff
@@ -3,67 +3,31 @@ title: Web deployment
 description: Learn how to build your Dart web app for production deployment.
 ---
 
-Deploying a Dart web app is similar to deploying any other web app:
-you first need to compile the app to JavaScript.
-This page describes how to compile your app—with
-tips for making it smaller and faster—and
-points you to resources for serving the app.
+Deploying a Dart web app works like deploying any other web app.
+This page describes how to compile your app, tips for making it smaller
+and faster, and points you to resources for serving the app.
 
 ## Building your app {#compiling-to-javascript}
 
-Use the webdev tool to build your app,
-compiling it to JavaScript and generating all the assets
-you need for deployment.
-When you build using [dart2js][],
+Use the `webdev` tool to build your app. It compiles Dart to JavaScript
+and generates all the assets you need for deployment.
+When you build using the production mode of the compiler,
 you get a JavaScript file that's reasonably small,
-thanks to the dart2js compiler's support for tree shaking.
+thanks to the compiler support for tree shaking.
 
 With a little extra work, you can make your deployable app
 [smaller, faster, and more reliable](#make-your-app-smaller-faster-and-more-reliable).
 
 ### Compile using webdev
 
-[Use the **webdev build** command][build] to create
-a deployable version of your app.
-Here's what happens when you use webdev with dart2js
-and the `--output build` option:
-
-* Deployable files appear under your app's `build/web` directory.
-* dart2js compiles your app to JavaScript, saving the result
-  in the file `build/web/main.dart.js`.
-
-For more information, see the [documentation for webdev][webdev].
-
-### Use dart2js flags to produce better JavaScript
-
-Google's apps often use the following [dart2js options](/tools/dart2js#options):
-
-- `--minify`
-- `--trust-primitives`
-- `--trust-type-annotations`
-
-**Test your apps before deploying with these options!**
-
-- The `--trust-primitives` option can have unexpected results
-  (even in well-typed code) if your data isn't always valid.
-
-For more information, see the [documentation for dart2js][dart2js].
-
-{{site.alert.important}}
-  Make sure your app has good test coverage
-  before you use either of the `--trust-*` options.
-  If some code paths aren't tested,
-  your app might run with dartdevc but
-  misbehave when compiled using dart2js.
-{{site.alert.end}}
-
-You can specify dart2js options in a build config file,
-as described in the [build_web_compilers page.][build_web_compilers]
+[Use the `webdev build` command][build] to create a deployable version
+of your app. This command converts your code to JavaScript and saves
+the result as `build/web/main.dart.js`.
```

Comment:
I agree with most of the deletions here, but it may be worth including also a link to the dart2js options (I wouldn't document them individually here, but refer to the fact that under the hood `webdev build` invokes `dart compile js`, so all options available there can also be used to configure a `webdev build`.